src/js/controllers/index.js

```diff
@@ -948,6 +948,25 @@ angular.module('copayApp.controllers').controller('indexController', function($r
     }, 100);
   };
 
+  self.filter = function(search) {
+    if (lodash.isEmpty(search)) return self.txHistory;
+    var result = lodash.filter(self.txHistory, function(tx) {
+      return lodash.includes(tx.amountStr, search) ||
+        lodash.includes(tx.message, search) ||
+        lodash.includes(self.addressbook[tx.addressTo], search) ||
+        lodash.includes(tx.addressTo, search) ||
+        lodash.isEqual(formatDate(new Date(tx.time * 1000)), search);
+    });
+    return result;
+  }
+
+  function formatDate(date) {
```

Comment:
Is this function used outside of `self.filter`?. If not, I think it should be inside `self.filter`.